core/templates/services/context.service.spec.ts

```diff
@@ -183,6 +238,20 @@ describe('Context service', () => {
       expect(ecs.getEntityType()).toBe('question');
       expect(ecs.getEntityId()).toBe('questionId');
     });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        spyOn(urlService, 'getPathname').and.returnValue('/skill_editor/123');
+        expect(ecs.getPageContext()).toBe('skill_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(true);
+      });
+
+    it('should affirm the page context to be exploration player or editor',
```

Comment:
Split this into two different tests -- one for player and another for editor.

Reply:
Done!